microshift_running_apps/microshift-operators-olm.adoc

```diff
@@ -0,0 +1,64 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="olm-with-microshift"]
+= Using Operator Lifecycle Manager with {microshift-short}
+include::_attributes/attributes-microshift.adoc[]
+:context: microshift-operators-olm
+
+toc::[]
+
+OLM is used in {microshift-short} for installing and running optional link:https://access.redhat.com/documentation/en-us/openshift_container_platform/{ocp-version}/html/architecture/control-plane#olm-operators_control-plane[add-on Operators].
+
+* Cluster Operators as applied in {ocp} are not used in {microshift-short}.
+* You can add your own catalogs for the Operators you want to use with your applications.
```

Comment:
Actualy you "*have* to add your own catalog". There is no catalog added per default. This is to ensure minimum resource usages, as catalog size has an impact on resource usage.

Reply:
Scooped up these suggestions in latest. Restructure is a bit of a rough cut, I'm sure we'll have more.